app/views/category_views.mli

```diff
@@ -0,0 +1,14 @@
+val creation_form : string -> [> Html_types.form ] Tyxml_html.elt
+val all : Models.Category.t list -> [> Html_types.table ] Tyxml_html.elt
+
+(** a view for listing all categories **)
+val categories :
+  ?flash_info:Models.Flash_info.t ->
+  ?user:Models.User.t ->
+  Models.Category.t list -> [> Html_types.html ] Tyxml_html.elt
+
+(** a view for listing all categories grouped by their number of topics **)
+val categories_by_topics_count :
```

Comment:
Names was not clear (and having `categories` as a prefix in a module called `Category` seems a little bit weird).
What is the difference between `all` and `categories`?

Reply:
Your right its weird, and i forgot to remove categories, this view is useless now. (it was my previous view).

Reply:
so, can i rename "category_line" too ? (which was already there before my PR), to "line"